Segment/Classes/SEGWebhookIntegration.h

```diff
@@ -0,0 +1,13 @@
+#import "SEGIntegration.h"
+#import "SEGIntegrationFactory.h"
+#import "SEGHTTPClient.h"
+
+NS_SWIFT_NAME(WebhookIntegrationFactory)
+@interface SEGWebhookIntegrationFactory : NSObject <SEGIntegrationFactory>
+
+@property(nonatomic, copy) NSString *name;
+@property(nonatomic, copy) NSString *webhookUrl;
```

Comment:
Are either of these properties nullable by design?

Reply:
both should be non-nullable and required